airflow/www_rbac/security.py

```diff
@@ -206,18 +206,14 @@ def init_role(self, role_name, role_vms, role_perms):
         if not role:
             role = self.add_role(role_name)
 
-        if len(role.permissions) == 0:
```

Comment:
do we have permissions for the role and so we always skip this snippet ? If its working for admin that does mean we never get to this block for admins too right ?

Reply:
yea admin has their own update function which is basically the same logic (recreate the role every time) https://github.com/Nextdoor/airflow/blob/nd_1.10.15/airflow/www_rbac/security.py#L448

Reply:
cool. can we add a comment here ?